File: Camera/Camera/FilterCollectionViewCell.m
//
//  FilterCollectionViewCell.m
//  Camera
//
//  Created by Joe E. on 10/19/15.
//  Copyright © 2015 Joe E. All rights reserved.
//

#import "FilterCollectionViewCell.h"
#import "ImageEditing.h"

@implementation FilterCollectionViewCell

- (void)didMoveToWindow {
//    [self filterImage];
    self.filterImageView.image = filterImage(self.originalImage, self.filterName);
    
}

- (void)prepareForReuse {
//    [self filterImage];
    self.filterImageView.image = filterImage(self.originalImage, self.filterName);
}

- (void)filterImage {
   // [self filterImage];
    
}

- (void) runFilterInBackground {

}


@end

File: Selfie/Selfie/FilterCollectionViewCell.swift
//
//  FilterCollectionViewCell.swift
//  Selfie
//
//  Created by Joe E. on 11/2/15.
//  Copyright © 2015 Joe E. All rights reserved.
//

import UIKit

class FilterCollectionViewCell: UICollectionViewCell {
    
    //MARK: - Properties
    var originalImage = UIImage()
    var filterName = String()
    
    //MARK: @IBOutlets
    @IBOutlet var filterImageView: UIImageView!
    
    override func didMoveToWindow() {
        super.didMoveToWindow()
//        filterImageView.image = filterImage(self.originalImage, filterName as String)
    }
    
    func filterImage() {
    
    }
    
    func runFilterInBackground() {
        
        
    }
    
}
